### PR TITLE
Add vangelists

### PR DESCRIPTION
**Pull request by vangelists**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [x] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [x] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/vangelists/vangelists.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Vangelis Tsiatsianas
+institution: Computer Science Department - University of Crete
+profile_pic: vangelists.png
+quote: '"'
```
Comment (juanpflores) [inline _data/vangelists/vangelists.md:5]: We were expecting some more content here 🤔

Comment (vangelists) [reply _data/vangelists/vangelists.md:5]: Could not find a quote! 😀

Comment (vangelists) [reply _data/vangelists/vangelists.md:5]: Could you use the literal quote as it is or is a phrase absolutely required? 🙂

Comment (juanpflores) [reply _data/vangelists/vangelists.md:5]: Yep, it is a requirement

Comment (vangelists) [reply _data/vangelists/vangelists.md]: Updated! 🙂

File: _data/vangelists/vangelists.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Vangelis Tsiatsianas
+institution: Computer Science Department - University of Crete
+profile_pic: vangelists.png
+quote: Bazinga! 💩
```
Comment (Mohammed-Shoaib) [inline _data/vangelists/vangelists.md:5]: Remove the 💩 (poop emoji) as it is a non-ascii character.

Comment (vangelists) [reply _data/vangelists/vangelists.md:5]: Hi Mohammed,

@juanpflores has mentioned that emojis are allowed: https://github.com/education/graduation/issues/348#issuecomment-631776551.

**Comment by vangelists** [comment]
Thank you for the additional review and the congrats, Mohammed! 🙂

**Comment by vangelists** [comment]
> Hey @vangelists,
> 
> Thanks for letting me know! I had originally assumed the character set was limited to _only_ ascii. Sorry about that! Your pull request looks good 👌
> 
> I confirm the following is correct:
> 
> 1. Checklist is complete. ✔️
> 2. Pull request only modified submission files (2 files) 📂
> 3. Profile picture is squared and has at least 544 pixels 🔳
> 
> I wish you all the best in your future endeavors! 🦄
> 
> Best,
> Mohammed Shoaib

Thank you for the additional review and the congrats, Mohammed! 🙂

**Comment by Mohammed-Shoaib** [comment]
Thanks you so much @vangelists! :heart:

Your pull request should be merged soon, till then sit back, relax, and enjoy :relaxed:

**Pull request merged**